src/ui/components/DownloadsManagerView/index.tsx

```
@@ -237,15 +238,25 @@ const DownloadsManagerView = () => {
+          paddingInline={2}
+          style={{ color: '#C0C0C0' }}
+        >
           <SelectLegacy
             value={mimeTypeFilter}
             placeholder={t('downloads.filters.mimeType')}
             options={mimeTypeOptions}
             onChange={handleMimeFilter}
           />
         </Box>
-        <Box display='flex' flexGrow={3} flexShrink={3} paddingInline={2}>
+        <Box display='flex'
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)